.github/workflows/ci-stable-release.yml

```diff
@@ -65,3 +68,43 @@ jobs:
           base: main
           signoff: true
           delete-branch: true
+
+  build-and-push-controller-versioned-release:
```

Comment:
We don't need this workflow

Ideally it should add simple as 
https://github.com/kubearmor/KubeArmor/blob/ea80cd58d2621914dfdcd9e17cd176172d8bb69f/.github/workflows/ci-stable-release.yml#L40

Adding another line below this 
```
regctl image copy kubearmor/kubearmor-controller:$STABLE_VERSION kubearmor/kubearmor-controller:stable --digest-tags 
```

This would work because `kubearmor/kubearmor-controller:vX.X` already exists and is pushed as part of `ci-latest-release`

Reply:
so with these changes we'll be releasing `kubearmor/kubearmor-controller:vX.X` along with each release of the KubeArmor not only the stable release right!